core/Updater/Migration/Custom.php

```diff
@@ -20,16 +20,18 @@ class Custom extends Migration
 {
     private $callback;
     private $toString;
+    private $args;
 
-    public function __construct($callback, $toString)
+    public function __construct($callback, $toString, $args = null)
     {
         $this->callback = $callback;
         $this->toString = $toString;
+        $this->args = $args;
     }
 
     public function exec()
     {
-        call_user_func($this->callback);
+        call_user_func($this->callback, $this->args);
```

Comment:
Shouldn't we allow passing multiple parameters here? The name `args` already sounds like that would be possible, but in fact your can only pass one argument with it.
I guess it would make more sense to use this here:
```suggestion
        call_user_func_array($this->callback, $this->args);
```